### PR TITLE
Fixup: Add a link to mender docs for enabling wifi in hosted image

### PR DESCRIPTION
**Pull request by oleorhagen**
Changelog: Title

Signed-off-by: Ole Petter <ole.orhagen@cfengine.com>

### CODE REVIEW

File: src/js/components/help/connecting-devices/raspberry-pi-3.js [hunk 1]
```diff
@@ -92,6 +92,9 @@ var RaspberryPi =  createReactClass({
         <p>Specifically, your device needs to be able to create outgoing TCP connections to hosted.mender.io and s3.amazonaws.com, both on port 443 (TLS).</p>
         <p>As the Mender client does not listen to any ports there are no incoming connections.</p>
 
+        <h4>Enabling wifi in the image</h4>
+        <p>Enabling wifi can be done, by following the instructions in: <a>https://docs.mender.io/development/getting-started/deploy-to-physical-devices</a></p>
```
Comment (estenberg) [inline src/js/components/help/connecting-devices/raspberry-pi-3.js]: * Would remove "development" from the URL, which maps to the latest stable instead
* Would also link directly to the section in the docs "Wifi connectivitiy", after https://github.com/mendersoftware/mender-docs/pull/409 is merged
* They also need to get the mender-artifact utility, but suppose that is ok given it is on the same page. Or change to first get mender-artifact, then follow the Wifi connectivity doc.

Comment (oleorhagen) [reply src/js/components/help/connecting-devices/raspberry-pi-3.js]: mender-artifact cp is still not in any released version.

Comment (estenberg) [reply src/js/components/help/connecting-devices/raspberry-pi-3.js]: Ahh, right. Then keep development and make a reminder to remove it once 1.6 is out :)

Comment (oleorhagen) [reply src/js/components/help/connecting-devices/raspberry-pi-3.js]: done

Comment (oleorhagen) [reply src/js/components/help/connecting-devices/raspberry-pi-3.js]: github should have triggered reminders

**Comment by estenberg** [comment]
Nice, good to go from my side (we need to get those docs live first though).

**Comment by oleorhagen** [comment]
So u do not want to link to the develop-docs?

**Pull request merged**